examples/complex-chart/complex-chart-example.js

```diff
@@ -77,11 +77,18 @@ export default class Example extends React.Component {
     ]
   }
 
+  componentWillMount() {
+    const {crosshairValues, series} = this;
```

Comment:
I think this is a legacy thing, you can reset the changes of the `componentWillMount`.